src/signtool/IBM_Socket.cpp

```diff
@@ -0,0 +1,516 @@
+#include <sys/types.h>
```

Comment:
No (C) header.

This code has enough problems that I'm not even going to go through it. I've gone over it before.